src/veh_type.cpp

```diff
@@ -263,7 +263,7 @@ void vpart_info::load_wheel( cata::optional<vpslot_wheel> &whptr, JsonObject &jo
 
 void vpart_info::load_workbench( cata::optional<vpslot_workbench> &wbptr, JsonObject &jo )
 {
-    vpslot_workbench wb_info;
+    vpslot_workbench wb_info{};
```

Comment:
Shouldn't this be fixed in `vpart_info::load_wheel()` and `vpart_info::load_engine()` as well?

Reply:
Technically it doesn't matter for those because they have sufficient non-static member data initializers that it's not an issue.  But consistency would be good and it's a risk for the future, so I've added them.